HTTPServer.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
We're not using `systemd` yet, so you can remove this file until we get back to that.

Reply:
Agreed